src/engine/DrawManager.java

```diff
@@ -981,6 +981,16 @@ private Color blinkingColor(String color) {
 			Color gray = new Color(RGB, RGB, RGB);
 			return gray;
 		}
+		if (color == "RANDOM") {
```

Comment:
After implementing the feature, I noticed that it was working similar to the ```color == "HIGH_SCORES"```. However, it is expected that it will provide a new feeling in that the color can be **specified**. Thank you.